src/appl/jj2lsj90/jj2lsj_code.f90

```diff
@@ -1684,14 +1686,14 @@ SUBROUTINE jj2lsj
             call convrt_double(asf_set_LS%csf_set_LS%csf(iw(1))%totalJ,string_CNUM,string_l)
             print 16, asf_set_LS%asf(level)%level_No,string_CNUM(1:string_l), &
                    asf_set_LS%csf_set_LS%csf(iw(1))%parity,                   &
-                   (weights2(j),iw(j),J=1,nocsf_min)
+                   (weights2(j),iw(j),J=1,5)
```

Comment:
Is it a good idea to hard-code this `5` in here? Why does this need to be changed away from `nocsf_min`?

Reply:
Should this here also use sum_nocsf_min by any chance? As I said, it looks strange that we'd hard-code the 5 here.

Reply:
Not sure, I asked @gaigalas about it - let's see what he says. I'm sure there's a solid reason for it to be 5.

Reply:
Just emailed with Gedminas, and he says there is a good reason for this, but a bit complicated and that we'll discuss it over video chat next time we meet. So for me, that's good enough. But great that we double checked it. Thanks M!